extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php

```diff
@@ -19,7 +19,8 @@ class StaffWelcomePoster {
 		'zh-hant' => 56584     // Ffaarr
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "zh-hant" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)